Doc/c-api/float.rst

```diff
@@ -76,3 +76,83 @@ Floating Point Objects
 .. c:function:: double PyFloat_GetMin()
 
    Return the minimum normalized positive float *DBL_MIN* as C :c:type:`double`.
+
+
+Pack and Unpack functions
+=========================
+
+The :mod:`struct` and :mod:`pickle` (at least) modules need an efficient
+platform- independent way to store floating-point values as byte strings. The
+Pack routines produce a bytes string from a C :c:type:`double`, and the Unpack
+routines produce a C :c:type:`double` from such a bytes string. The suffix
+(2, 4 or 8) specifies the number of bytes in the bytes string.
+
+On platforms that appear to use IEEE-754 formats these functions work by
+copying bits. On other platforms, the formats the 4- byte format (32-bit) is
+identical to the IEEE-754 single precision format, and the 8-byte format
```

Comment:
To make this unambiguous, I'd suggest `the IEEE 754 binary32 "single precision" format`, and similarly for double precision. It's reasonably easy to _guess_ that "IEEE 754 single precision" means the binary32 format, but it's not totally unambiguous, and it would be better not to have to guess. (IEEE 754 specifies 4-byte decimal formats, too, and the standard itself never uses the terms "single precision" or "double precision".)